server/routes/api/delete/deleteUserHandler.ts

```diff
@@ -9,49 +9,49 @@ export default class Handler {
     this.dbHandler = new DBHandler();
   }
 
-  parentValidation = async(parentId: number): Promise<Validation> => {
+  parentValidation = async (parentId: number): Promise<Validation> => {
     const parentProfile = await this.dbHandler.getParent(parentId);
     if (!parentProfile.length) {
-      return { isValid: false, message: 'Incorrect id' }
+      return { isValid: false, message: "Incorrect id" };
     }
 
-    if (parentProfile[0].endTimestamp !== END_TIMESTAMP) {
-      return { isValid: false, message: 'This user is not active' }
+    if (Number(parentProfile[0].endTimestamp) !== END_TIMESTAMP) {
```

Comment:
i beleave it should be != instead of !==
When you test it does it work? (it didn't work to me)